packages/dscc-gen/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "es2017",
     "outDir": "build",
+    "sourceMap": true,
```

Comment:
Why do we want this added?

Reply:
I don't remember - I think it was some vscode autocomplete. will remove